packages/p2p/jest.config.js

```diff
@@ -32,5 +32,5 @@ module.exports = {
         '<rootDir>/coverage/lcov-report',
         '<rootDir>/dist',
     ],
-    transformIgnorePatterns: ['/node_modules/(?!@sendbird/chat).+\\.js$'],
+    transformIgnorePatterns: ['/node_modules/(?!(@sendbird/chat|@simplewebauthn/browser)).+\\.js$'],
```

Comment:
why do we need this change in p2p package?

Reply:
there was an error during running all tests. so I had to add it to all related configs.
as I understood jest runs tests in node.js environment with CommonJS, and not ESM. so it argues on this library, because it uses ESM.
so the only way works here (at least from what I've tried) is to configure transformIgnorePatterns